src/Controllers/CategoryController.php

```diff
@@ -44,6 +44,19 @@ public function showCategory(
         $lang = $sessionService->getLang();
         $webstoreId = pluginApp(Application::class)->getWebstoreId();
 
+        /** @var Request $request */
+        $request = pluginApp(Request::class);
+        if ($request->get('isContentBuilder', false) && $request->get('contentType', '') === 'singleitem')
+        {
+            return $this->renderTemplate(
```

Comment:
anstatt hier das Item-Template zu rendern, würde ich eher `ItemController::showItem()` aufrufen, um evtl. nebeneffekte des Controllers auch im ShopBuilder zu nutzen.

Reply:
done. habe auch den code in ceres dementsprechend angepasst